framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -986,7 +1039,8 @@ private void handleExpr(AnnotationBuilder builder, String name,
                     builder.setValue(name, arr);
                 }
             } else {
-                ErrorReporter.errorAbort("StubParser: unhandled annotation attribute type: " + expr + " and expected: " + expected);
+                ErrorReporter.errorAbort("StubParser: unhandled annotation attribute type: " + expr +
+                        " and expected: " + expected);
```

Comment:
I prefer to put operations at the beginning of a line rather than at the end of the previous one -- I find it makes the code easier to read, since one can tell immediately what operation is being performed.